dspace/config/dspace.cfg

```diff
@@ -163,6 +163,10 @@ mail.allowed.referrers = ${dspace.hostname}
 # This is especially useful for development and test environments where production data is used when testing functionality.
 #mail.server.disabled = false
 
+# Message headers which may be set by a message template.
+mail.message.headers = subject
+mail.message.headers = charset
```

Comment:
Are there additional possible values here? Could we enhance the comment description to describe what "subject" and "charset" means here?  I don't quite understand these configurations myself without digging further into the code behind them.  (Obviously this should also be in the Documentation, but it'd be good to provide better hints in the dspace.cfg as to how to configure `mail.message.headers`)

Reply:
You can set any header, if its name is mentioned in this list.  'subject' and 'charset' are special in that they provide values for respectively a header and a header-parameter which would always be generated.  This capability already existed, but now uses Velocity properties instead of parsing stuff out of comments.

I've augmented the commentary in dspace.cfg as requested.  The Official Documentation for this should be updated (or written?) too.